Doc/source/optional.rst

```diff
@@ -1,142 +1,112 @@
-:orphan:
-
 Optional Dependencies
 =====================
 
-The fonttools PyPI distribution also supports so-called "extras", i.e. a
-set of keywords that describe a group of additional dependencies, which can be
-used when installing via pip, or when specifying a requirement.
-For example:
+The FontTools PyPI distribution supports "extras", which are groups of additional dependencies that can be installed alongside FontTools via pip to enable specific features. For example:
```

Comment:
I know that there is already a lack of total consistency in the docs regarding the capitalization of "FontTools" when it is used as the name of the project (versus, for example, referring to the modules). I see that this PR uses "FontTools" here, but uses "fontTools" in index.rst. Might as well be consistent within the PR, IMO.

Personally, I think starting with the capital F is the way to go, both because it more clearly differentiates between "project name" and "token" use cases, but also because of the project logo....

If there are other strong opinions, though, I'm happy to hear what they all are.